core/iwasm/fast-jit/fe/jit_emit_function.c

```diff
@@ -144,6 +144,105 @@ post_return(JitCompContext *cc, const WASMType *func_type, JitReg first_res)
     return false;
 }
 
+bool
+jit_compile_op_call_draft(JitCompContext *cc, uint32 func_idx, bool tail_call)
+{
+    WASMModule *wasm_module = cc->cur_wasm_module;
+    WASMFunctionImport *func_import;
+    WASMFunction *func;
+    WASMType *func_type;
+    JitFrame *jit_frame = cc->jit_frame;
+    JitReg native_ret;
+    JitReg fast_jit_func_ptrs, jitted_code = 0;
+    uint32 jitted_func_idx;
+
+    /* new var */
+    const char *signature = NULL;
+
+    if (func_idx >= wasm_module->import_function_count) {
```

Comment:
How about moving these codes (L163 to L168) to the next else branch (after L195)?

Reply:
OK, I will check the code.